resources/newssite/news-next/src/context/data-context.js

```diff
@@ -1,9 +1,24 @@
 import { createContext, useContext } from "react";
+
 import { content as contentEn } from "@/data/en/content";
+import { content as contentJp } from "@/data/jp/content";
+import { content as contentAr } from "@/data/ar/content";
+
 import { settings as settingsEn } from "@/data/en/dialog";
+import { settings as settingsJp } from "@/data/jp/dialog";
+import { settings as settingsAr } from "@/data/ar/dialog";
+
 import { footer as footerEn } from "@/data/en/footer";
+import { footer as footerJp } from "@/data/jp/footer";
+import { footer as footerAr } from "@/data/ar/footer";
+
 import * as buttonsEn from "@/data/en/buttons";
+import * as buttonsJp from "@/data/jp/buttons";
+import * as buttonsAr from "@/data/ar/buttons";
+
 import * as linksEn from "@/data/en/links";
+import * as linksJp from "@/data/jp/links";
+import * as linksAr from "@/data/ar/links";
```

Comment:
It is probably not scalable to import all the localized content like this because then everything ends up in the bundle, instead we probably want to use something like async imports or `fetch`. But maybe this is good enogh for us...

Reply:
yes, it's not for production usage 😄 
I wanted to avoid any fetch of content and for our purpose it works.